Views/backend/connect/view/import/own_categories.js

```diff
@@ -150,6 +150,13 @@ Ext.define('Shopware.apps.Connect.view.import.OwnCategories', {
             dock: 'top',
             ui: 'shopware-ui',
             items: [
+                {
+                    xtype: 'button',
+                    iconCls: 'sprite-minus-circle-frame',
+                    action: 'deactivateCategory',
+                    margin: '0 10px 0 0',
+                    text: '{s name=import/action/deactivate_category}Deactivate categories{/s}'
```

Comment:
Why "Deactivate categories" ? I think it should be "Deactivate category", because only one category can be deactivated ->
```
 params: {
  categoryId: selected[0].get('id')
 },
```